javascript/portfolio.js

```diff
@@ -14,17 +15,40 @@ function PortfolioObject(portData){
 //     portObjectArray.push(new PortfolioObject(portfolioData[i]));
```

Comment:
Since you are doing this somewhere else, this is corpse code. Please delete it before merging into master.